File: app/src/main/java/com/lugo/manueln/socialapp/Interfaces/loadfragments.java
package com.lugo.manueln.socialapp.Interfaces;

import com.lugo.manueln.socialapp.Views.AlbumsFragment;
import com.lugo.manueln.socialapp.Views.MenuFragment;
import com.lugo.manueln.socialapp.Views.PhotosFragment;
import com.lugo.manueln.socialapp.Views.PostCompleteFragment;
import com.lugo.manueln.socialapp.Views.PostsFragment;

public interface loadfragments extends PostsFragment.OnFragmentInteractionListener,PostCompleteFragment.OnFragmentInteractionListener,AlbumsFragment.OnFragmentInteractionListener,PhotosFragment.OnFragmentInteractionListener, MenuFragment.OnFragmentInteractionListener {
}

File: app/src/main/java/com/lugo/manueln/socialapp/utils/loadfragments.kt
package com.lugo.manueln.socialapp.utils

import com.lugo.manueln.socialapp.presentation.MenuF.MenuFragment
import com.lugo.manueln.socialapp.presentation.Views.PostComplete.View.PostCompleteFragment
import com.lugo.manueln.socialapp.presentation.Views.Posts.View.PostsFragment

interface loadfragments : PostsFragment.OnFragmentInteractionListener, PostCompleteFragment.OnFragmentInteractionListener, MenuFragment.OnFragmentInteractionListener
